__tests__/components/forecast-summaries.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import ForecastSummary from '../../src/components/forecast-summary';
+import ForecastSummaries from '../../src/components/forecast-summaries';
+
+describe('Forecast Summaries', () => {
+  const forecasts = [
+    {
+      date: 123,
+      description: 'date1',
+      icon: 'icon1',
+      temperature: {
+        max: 999,
+      },
+    },
+
+    {
+      date: 456,
+      description: 'date2',
+      icon: 'icon2',
+      temperature: {
+        max: 777,
+      },
+    },
+  ];
+
+  it('renders the correct number of ForecastSummary components', () => {
+    const wrapper = Enzyme.shallow((
+      <ForecastSummaries forecasts={forecasts} />
+    ));
+    expect(wrapper.find(ForecastSummary).length).toEqual(2);
```

Comment:
Try and avoid magic numbers. If you ever needed to add another forecast to your list it would break this test. Try `forecasts.length` instead.